command/pr.go

```diff
@@ -80,6 +84,13 @@ var prReopenCmd = &cobra.Command{
 	RunE:  prReopen,
 }
 
+var prMergeCmd = &cobra.Command{
+	Use:   "merge [number | url] [--rebase | --merge | --squash]",
```

Comment:
I wasn't sure how to document the idea of a mutually exclusive optional variable.
`[{number | url}]` seems like it might be correct, but `[number | url]` is easier to read and still seems unambiguous to me. It's hard to tell from command line docs what the correct syntax is.